src/Runner.cpp

```diff
@@ -1,3 +1,4 @@
+#pragma clang diagnostic ignored "-Wunused-parameter"
```

Comment:
Why do you need this pragma? You already suppress the warning in CMakeLists.txt as I see.